test/server/hot-option.test.js

```diff
@@ -34,6 +34,35 @@ describe("hot option", () => {
     });
   });
 
+  describe("simple config with hot: false", () => {
+    beforeAll(async () => {
+      const compiler = webpack(config);
+
+      server = new Server(
+        {
+          port,
+          hot: false,
+        },
+        compiler
+      );
+
+      await server.start();
+
+      req = request(server.app);
+    });
+
+    afterAll(async () => {
+      await server.stop();
+    });
+
+    it("should not include hot script in the bundle", async () => {
+      const response = await req.get("/main.js");
+
+      expect(response.status).toEqual(200);
+      expect(response.text).not.toContain("webpack/hot/dev-server.js");
+    });
+  });
```

Comment:
Let's add test for open browser, we should see no output for hot log

Reply:
I think it is already present - 

https://github.com/webpack/webpack-dev-server/blob/19b3678205d2408199e3f687f68fdf1e31e8d5e8/test/e2e/hot-and-live-reload.test.js#L52-L58

https://github.com/webpack/webpack-dev-server/blob/19b3678205d2408199e3f687f68fdf1e31e8d5e8/test/e2e/hot-and-live-reload.test.js#L67-L73

Reply:
hm, looks like duplicate test, but here we test `webpack/hot/dev-server.js`, I am fine with this